apps/modernization-ui/src/design-system/select/multi/theme.ts

```diff
@@ -0,0 +1,9 @@
+import { Theme } from 'react-select';
+
+const theme = (existing: Theme) => ({
+    ...existing,
+    borderRadius: 2,
+    colors: { ...existing.colors, primary: '#DEE7F5', primary25: '#' }
```

Comment:
Question: I didn't see `#DEE7F5` in our styles/colors. Should that value be added?